mkdocs.yml

```diff
@@ -75,7 +76,9 @@ nav:
         - Overview: Cakeshop/Overview.md
         - Getting Started: Cakeshop/Getting started.md
         - Cakeshop FAQ: Cakeshop/Cakeshop FAQ.md
-    - Product Roadmap: roadmap.md         
+    - Quorum Features:
```

Comment:
As all features are in the 1st level, DNS is deserved in the first level too.
Otherwise we could move others to `Enterprise Features` menu. But i prefer the former.

Reply:
It was intended to be a place to put features that don't fit the "main" categories. Adding them to the top level would create too much clutter, but happy to consider alternatives.